syncd/Syncd.cpp

```diff
@@ -204,9 +204,9 @@ void Syncd::performStartupLogic()
 {
     SWSS_LOG_ENTER();
 
-    // ignore warm logic here if syncd starts in Mellanox fastfast boot mode
+    // ignore warm logic here if syncd starts in fast-boot or Mellanox fastfast boot mode
 
-    if (m_isWarmStart && (m_commandLineOptions->m_startType != SAI_START_TYPE_FASTFAST_BOOT))
+    if (m_isWarmStart && m_commandLineOptions->m_startType != SAI_START_TYPE_FASTFAST_BOOT && m_commandLineOptions->m_startType != SAI_START_TYPE_FAST_BOOT)
```

Comment:
It may be better if this condition checks for positive match for `SAI_START_TYPE_WARM_BOOT` instead of a negative match for fastfast and fast reboot.

Reply:
In this case the assignment in line 214: "m_commandLineOptions->m_startType = SAI_START_TYPE_WARM_BOOT;" is redundant, not?
Should I change to the positive match and remove the redundancy?

Reply:
What do you think?